docker-compose.yml

```
@@ -0,0 +1,39 @@
+services:
+  isaac-sim:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+      args:
+        USER_ID: ${USER_ID:?}
+        GROUP_ID: ${GROUP_ID:?}
+        SHARED_GROUP_ID: ${SHARED_GROUP_ID}
+        DOCKER_USERNAME: ${DOCKER_USERNAME:?}
+    image: ${USERNAME:?}/nvsynth
+    user: ${USER_ID:?}:${GROUP_ID:?}
+    volumes:
+      - .:/nvsynth
+      # food data volume
+      - ${DATASET_PATH:-/dev/null}:/food
```

If we're mounting the food directory to `/food`, then it would be nice if the entrypoint could add a symlink from `/food` -> `/nvsynth/assets/food`. The instructions in the original ReadMe ask the user to create this symlink manually before running the apps, but since everything is containerized, we should just take care of this for them.

The reason why I want the food assets to show up in the repo directory (at least symbolically) is so that you can visualize the textures and USD files without having to add a new (external) folder to the workspace :D

---

Actually nvm, creating a symlink inside the container is a bad idea since the link will still point to a location inside the container once it exists (which breaks the dir on the host filesystem). Instead, let's change this so that it mounts it directly into the right directory
```suggestion
      - ${DATASET_PATH:-/dev/null}:/assets/food
```